Rlabkey/R/labkey.transform.R

```diff
@@ -0,0 +1,50 @@
+
+labkey.transform.readRunPropertiesFile <- function(run.info.path)
+{
```

Comment:
a little odd to have dots in our variable names, I think this is a departure from the style we typically use, suggest changing to runInfoPath.